src/api/app/controllers/webui/project_controller.rb

```diff
@@ -655,6 +655,8 @@ def load_project_info
 
     reqs = @project.open_requests
     @requests = (reqs[:reviews] + reqs[:targets] + reqs[:incidents] + reqs[:maintenance_release]).sort!.uniq
+    @incoming_requests = (reqs[:incidents] + reqs[:maintenance_release]).sort!.uniq
```

Comment:
`sort` + `uniq` is equivalent to `|` which is way more efficient. In Ruby 2.6, you can also use `union` (but we are using 2.5). More info and metrics: http://anamaria.martinezgomez.name/2018/12/21/ruby2_6.html